src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementProjectService.java

```diff
@@ -368,42 +363,100 @@ public TeamMember addProjectTeamMember(final Integer projectId, final String use
+      var dbEvent = getCurrentEvent(project.getProject());
+      // TODO make single call instead of 2
+      try {
+        var agreementDetails = agreementsService.getAgreementDetails(agreementNo);
+        var lotDetails = agreementsService.getLotDetails(agreementNo, project.getProject().getLotNumber());
+        projectPackageSummary.setAgreementName(agreementDetails.getName());
+        projectPackageSummary.setLotName(lotDetails.getName());
+      }catch (Exception e) {
+        //ignore for the moment, replace when single method to get all data from agreement service
+      }
       // TODO no value for Uri
       // projectPackageSummary.setUri(getProjectUri);
-      projectPackageSummary.setProjectId(dbProject.get().getId());
-      projectPackageSummary.setProjectName(dbProject.get().getProjectName());
-
-      /*
-       * TODO which field from Jaaggaer
-       * TenderStatus.fromValue(project.getTender().getTenderStatusLabel().substring(PROJECT_STATE.
-       * length()))
-       */
+      projectPackageSummary.setAgreementId(project.getProject().getCaNumber());
+      projectPackageSummary.setLotId(project.getProject().getLotNumber());
+      projectPackageSummary.setProjectId(project.getProject().getId());
+      projectPackageSummary.setProjectName(project.getProject().getProjectName());
+
+      var exportRfxResponse = jaggaerService.getRfx(dbEvent.getExternalEventId());
+      var status = findTenderStatus(dbEvent,exportRfxResponse);
       var eventSummary = tendersAPIModelUtils.buildEventSummary(dbEvent.getEventID(),
           dbEvent.getEventName(), Optional.ofNullable(dbEvent.getExternalReferenceId()),
-          ViewEventType.fromValue(dbEvent.getEventType()), null, ReleaseTag.TENDER,
-          Optional.empty());
+          ViewEventType.fromValue(dbEvent.getEventType()), status, ReleaseTag.TENDER,
+              Optional.ofNullable(dbEvent.getAssessmentId()));
       projectPackageSummary.activeEvent(eventSummary);
       return Optional.of(projectPackageSummary);
-    }
-    // TODO ignoring at the moment. What to do with these?
-    log.warn("Unable to project details in database for project id  {}, so ignoring.",
-        project.getTender().getTenderCode());
-    return Optional.empty();
+
   }
 
-  private Optional<ProcurementProject> getProject(final List<ProcurementProject> projects,
-      final String externalReferenceId) {
-    return projects.stream().filter(
-        procurementProject -> procurementProject.getExternalProjectId().equals(externalReferenceId))
-        .findFirst();
+  private TenderStatus findTenderStatus(ProcurementEvent dbEvent, ExportRfxResponse exportRfxResponse) {
+    var statusCode = exportRfxResponse.getRfxSetting().getStatusCode();
+    String eventType = dbEvent.getEventType();
+    // This logic is based on attached excel of SCAT-3671
+    //TODO: some scenarios are have ambiguous and need to check end to this logic
+    //TODO: can it be better?
```

Comment:
I think we could externalise this config into the `application.yml` and pull into config properties as `Map<String, Map<String, TenderStatus>>` I think - this might make it easier to maintain and remove the risk of fall-through bugs in these switch statements?